test/units/TEST-75-RESOLVED.sh

```diff
@@ -104,7 +109,17 @@ assert_in '_localdnsstub' "$(dig @127.0.0.53 -x 127.0.0.53)"
 assert_in '127.0.0.54' "$(dig @127.0.0.53 _localdnsproxy)"
 assert_in '_localdnsproxy' "$(dig @127.0.0.53 -x 127.0.0.54)"
 
+# Cleanup
```

Comment:
It's customary--and safer--to move cleanup code into a function set up as an `EXIT` handler (that is, by calling `trap cleanup-func EXIT`. It's usually done for setup code too.

Reply:
Good point, thanks! I will take a look at improving this.

Reply:
Done, however since the cleanup is a part of the test case (function), I couldn't trap it to trigger on EXIT, but on RETURN.